packages/components-vue/src/pages/OverviewPage.vue

```diff
@@ -355,35 +345,35 @@
       </div>
 
       <div class="playground light" title="should render default link-tile">
-        <PLinkTile :href="'#'" :label="'Some Label'" :description="'Default'">
+        <PLinkTile :description="'Default'" :href="'#'" :label="'Some Label'">
           <img
+            alt="Beach"
+            height="50"
             src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADIAAAAyAQMAAAAk8RryAAAABlBMVEUAAAD2vP9xXLiUAAAAAXRSTlMAQObYZgAAABxJREFUGNNjYOBgYGBhYKAZ/R8MDsD4Q5amkz8ASp4PtTYYQZIAAAAASUVORK5CYII="
             width="50"
-            height="50"
-            alt="Beach"
           />
         </PLinkTile>
       </div>
 
       <div class="playground light" title="should render default button-tile">
-        <PButtonTile :label="'Some Label'" :description="'Default'">
+        <PButtonTile :description="'Default'" :label="'Some Label'">
           <img
+            alt="Beach"
+            height="50"
             src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADIAAAAyAQMAAAAk8RryAAAABlBMVEUAAAD2vP9xXLiUAAAAAXRSTlMAQObYZgAAABxJREFUGNNjYOBgYGBhYKAZ/R8MDsD4Q5amkz8ASp4PtTYYQZIAAAAASUVORK5CYII="
             width="50"
-            height="50"
-            alt="Beach"
           />
         </PButtonTile>
       </div>
 
       <div class="playground light" title="should render default link-tile-model-signature">
         <!-- eslint-disable vue/no-deprecated-slot-attribute -->
-        <PLinkTileModelSignature :heading="'Some Heading'" :description="'Default'">
+        <PLinkTileModelSignature :description="'Default'" :heading="'Some Heading'">
           <img
+            alt="Beach"
```

Comment:
Is this alphabetical sorting of attributes helpful? I think so far everything was ordered by importance/relevance.

Reply:
It was caused by a Webstorm Setting, I will revert it and disable the sorting